engine/src/test/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessorTest.java

```diff
@@ -48,9 +73,30 @@ class EmailSubscriptionTypeProcessorTest {
     @InjectMock
     EmailAggregationRepository emailAggregationRepository;
 
+    @InjectMock
+    EmailSubscriptionRepository emailSubscriptionRepository;
+
+    @InjectMock
+    RecipientResolver recipientResolver;
+
+    @InjectMock
+    EmailSender sender;
+
+    @Inject
+    StatelessSessionFactory statelessSessionFactory;
+
     @Inject
     MicrometerAssertionHelper micrometerAssertionHelper;
 
+    @Inject
+    FeatureFlipper featureFlipper;
+
+    @BeforeEach
+    void beforeEach() {
+        featureFlipper.setUseTemplatesFromDb(false);
+        featureFlipper.setUseTemplatesFromDb(false);
```

Comment:
Is this what you meant to write?
```suggestion
        featureFlipper.setUseTemplatesFromDb(false);
        featureFlipper.setUseDefaultTemplate(false);
```

Reply:
Whoops - yes